lib/http/2/header/encoding_context.rb

```diff
@@ -77,10 +77,11 @@ class EncodingContext
         ["www-authenticate",            ""]
       ].each { |pair| pair.each(&:freeze).freeze }.freeze
 
-      STATIC_TABLE_BY_FIELD = STATIC_TABLE
-                              .each_with_object({})
-                              .with_index { |((field, value), hs), idx| (hs[field] ||= []) << [idx, value] }
-                              .each { |pair| pair.each(&:freeze).freeze }.freeze
+      STATIC_TABLE_BY_FIELD =
+        STATIC_TABLE
+        .each_with_object({})
+        .with_index { |((field, value), hs), idx| (hs[field.freeze] ||= []) << [idx, value.freeze].freeze }
```

Comment:
Ditto on freeze question

Reply:
same

Reply:
Do you need `value.freeze`? I thought just arrays needed to be.